docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,200 @@
+      alluxio://RM_hostname:19998/s3 s3a://<S3_BUCKET>/<S3_DIRECTORY>
+   ```
+
+   For other filesystem, please refer to [this site](https://www.alluxio.io/)
+
+6. Start Alluxio cluster.
+
+   Login Alluxio master node, and run
+
+   ``` bash
+   ${ALLUXIO_HOME}/bin/alluxio-start.sh all
+   ```
+
+## RAPIDS Configuration
+
+There are two ways to leverage Alluxio in RAPIDS.
+
+1. Explicitly specify alluxio path.
+
+   This may require user to change code.
+
+   Eg. Change below code
+
+   ``` scala
+   val df = spark.read.parquet("s3a://<S3_BUCKET>/<S3_DIRECTORY>/foo.parquet")
+   ```
+
+   to
+
+   ``` scala
+   val df = spark.read.parquet("alluxio://RM_hostname:19998/s3/foo.parquet")
+   ```
+
+2. Transparently replace in RAPIDS.
+
+   RAPIDS has added a configuration `spark.rapids.alluxio.pathsToReplace` which can allow RAPIDS
+   to replace the input file paths to alluxio paths transparently at runtime. So there is no any
+   code change for users.
+
+   Eg, at startup
+   ``` shell
+   --conf spark.rapids.alluxio.pathsToReplace="s3:/foo->alluxio://RM_hostname:19998/foo,gs:/bar->alluxio://RM_hostname:19998/bar"
+   ```
+
+   This configuration allows RAPIDS to replace any file paths prefixed `s3:/foo` to
+   `alluxio://RM_hostname:19998/foo` and `gs:/bar` to `alluxio://RM_hostname:19998/bar`
+
+3. Submit an application.
+
+   Spark driver and tasks will parse `alluxio://` schema and access Alluxio cluster by
+   `alluxio-${LATEST}-client.jar` which must be distributed across the all nodes where Spark drivers
+   or executors are running.
+
+   The Alluxio client jar must be in the classpath of all Spark drivers and executors in order for
+   Spark applications to access Alluxio.
+
+   We can specify it in the configuration of `spark.driver.extraClassPath` and
+   `spark.executor.extraClassPath`, but the simplest way is copy `alluxio-${LATEST}-client.jar`
+   into spark jars directory.
+
+   ``` shell
+   cp ${ALLUXIO_HOME}/client/alluxio-${LATEST}-client.jar ${SPARK_HOME}/jars/
+   ```
+
+   ``` shell
+   ${SPARK_HOME}/bin/spark-submit \
+      ...                          \
+      --conf spark.rapids.alluxio.pathsToReplace="REPLACEMENT_RULES" \
+      --conf spark.executor.extraJavaOptions="-Dalluxio.conf.dir=${ALLUXIO_HOME}/conf" \
+   ```
+
+## Alluxio troubleshoot
+This section will give some links about how to configure, tune Alluxio and some troubleshooting.
+- [Quick Start Guide](https://docs.alluxio.io/os/user/stable/en/overview/Getting-Started.html)
+- [Amazon S3 as Alluxio’s under storage system](https://docs.alluxio.io/os/user/stable/en/ufs/S3.html)
+- [Alluxio metrics](https://docs.alluxio.io/os/user/stable/en/reference/Metrics-List.html)
+- [Alluxio configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html)
+- [Running Spark on Alluxio](https://docs.alluxio.io/os/user/stable/en/compute/Spark.html)
+- [Performance Tuning](https://docs.alluxio.io/ee/user/stable/en/operation/Performance-Tuning.html)
```

Comment:
https://docs.alluxio.io/os/user/stable/en/operation/Performance-Tuning.html could you use this open source link instead of the enterprise edition link for consistency?